packages/database/lib/index.d.ts

```diff
@@ -167,9 +167,13 @@ export namespace FirebaseDatabaseTypes {
     child(path: string): Reference;
 
     /**
-     * The last part of the Reference's path.
-     * For example, "ada" is the key for https://<DATABASE_NAME>.firebaseio.com/users/ada.
-     * The key of a root Reference is null.
+     * Returns an `OnDisconnect` object - see
+     * {@link
+     *   https://firebase.google.com/docs/database/web/offline-capabilities
+     *   Enabling Offline Capabilities in JavaScript} for more information on how
+     * to use it.
+     *
+     * @return An `OnDisconnect` object .
```

Comment:
This does not seem correct?

The rest seem like good changes - typing the function return more strongly is a benefit I think

Reply:
so.. What can I do?

this is my first pull request and i am begginer in github..

Reply:
The documentation here in this part is saying it will return an `onDisconnect` object etc, but is connected to the `key` property. So the documentation is not valid and needs correction